.gitignore

```diff
@@ -1,8 +1,7 @@
 .vscode/
 pkg/
-node_modules/
-analysis/
-build/
-target/
+**/node_modules/
+**/build/
+**/target/
```

Comment:
Our `.dockerignore` file is just a link to the `.gitignore`. I thought the two used the same syntax but apparently not because docker was copying in the node_modules and other stuff. This should make it behave as intended.